apps/emqx_auth_mnesia/src/emqx_authn_mnesia.erl

```diff
@@ -224,26 +230,45 @@ import_users({PasswordType, Filename, FileData}, State, Opts) ->
 do_import_users([], _Opts) ->
     {error, empty_users};
 do_import_users(Users, Opts) ->
+    IncBucketFn = fun(Bucket, Acc) ->
+        N = maps:get(Bucket, Acc, 0),
+        maps:put(Bucket, N + 1, Acc)
+    end,
+    FoldFn = fun(User, Acc0) ->
+        Return = insert_user(User, Opts),
+        case Return of
+            {success, Ns} ->
+                PerNs0 = maps:get(per_ns, Acc0),
+                PerNs = maps:update_with(Ns, fun(N) -> N + 1 end, 1, PerNs0),
```

Comment:
Nit: this seems to be the same as `IncBucketFn`, m.b. rename `IncBucketFn -> IncKey` and use also here